README.md

```diff
@@ -173,11 +177,12 @@ Our `hello-snowman` app will read and interpret these messages and will inform u
 - in the other case (temperature above 0 degrees), sequence will return message `Snowman is melting! :(`
 
 To run this app, please execute this command from the root of our directory `node ./tools/stream-gen-tool/stream-gen.js <instance_id>`. In our case this would look like this:
```

Comment:
```suggestion
To run this app, please execute the following command from the root of our directory `node ./tools/stream-gen-tool/stream-gen.js <instance_id>`. In our case this would look like this:
```